mesen-s.json

```
@@ -0,0 +1,34 @@
+{
+    "homepage": "https://www.mesen.ca",
+    "description": "A high-accuracy NES and Famicom emulator and NSF player.",
+    "version": "0.4.0",
+    "license": "GPL-3.0",
+    "url": "https://github.com/SourMesen/Mesen-S/releases/download/0.4.0/Mesen-S.0.4.0.zip",
+    "hash": "CE4EE543DD3AAA4A10E8597D5AA4735FBE9252C8D679E5772DBEE827CFAA2D96",
```

Use lowercase hash for consistency with other manifests:

```suggestion
    "hash": "ce4ee543dd3aaa4a10e8597d5aa4735fbe9252c8d679e5772dbee827cfaa2d96",
```